wp-cli-templates/block-edit-js.mustache

```diff
@@ -0,0 +1,27 @@
+/**
+* External dependencies
+*/
+import { __ } from '@wordpress/i18n';
+import { Component, Fragment } from '@wordpress/element';
+
+/**
+* Internal dependencies
+*/
+/* @TODO add any needed dependency */
+
+import './editor.scss';
+
+class {{ className }}Edit extends Component {
+
+    render() {
+        const { attributes, setAttributes } = this.props;
+
+        return (
+            <Fragment>
+
```

Comment:
What do you think if we switch `Fragment` to `<p>You're editing the block!</p>` so that. they won't see just empty block in editor?